browser/speedreader/speedreader_tab_helper.h

```diff
@@ -41,8 +43,10 @@ class SpeedreaderBubbleView;
 class SpeedreaderTabHelper
     : public content::WebContentsObserver,
       public content::WebContentsUserData<SpeedreaderTabHelper>,
+      public PageDistiller,
```

Comment:
i don't quite follow why do we need to inherit it instead of having it as a field